libgraph/include/katana/PropertyGraph.h

```diff
@@ -870,6 +870,10 @@ class KATANA_EXPORT PropertyGraph {
     return node_iterator(node_id);
   }
 
+  Result<bool> IsNodeIndexPresent(const std::string& column_name);
+
+  Result<bool> IsEdgeIndexPresent(const std::string& column_name);
```

Comment:
The first method already exists: `HasNodePropertyIndex`. The second does not, but should be called `HasEdgePropertyIndex` and should be implemented like the node version.

Please look more carefully before reimplementing things.